.github/workflows/ci.yml

```diff
@@ -60,6 +60,11 @@ jobs:
           command: fmt
           args: -- --check
 
+      - name: cargo doc
+        uses: actions-rs/cargo@v1
+        with:
+          command: doc
```

Comment:
Will this error on warnings?

I suspect not and this is one of the several reasons why I dislike the `cargo` action (its also a dependency that does not pull its weight).

I'd recommend something more like what I run in my other projects: https://github.com/rust-cli/argfile/blob/main/.github/workflows/ci.yml#L68

Reply:
Thanks for the review! Is this what you had in mind? 5a5a130ce5ba83c40453a866d410aa8630de0a85

Reply:
That works!